syspurpose/setup.cfg

```diff
@@ -1,4 +1,11 @@
 [nosetests]
-where=./test/syspurpose
-verbose=1
+tests=test
+verbosity=2
 nocapture=True
+with-randomly=True
+with-coverage=True
+cover-html=True
+cover-html-dir=htmlcov
+cover-xml=True
+cover-erase=True
```

Comment:
I would add here: `cover-package=syspurpose` to see coverage only for syspurpose package.